portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/logging/CompanyLogging.testcase

```diff
@@ -22,75 +24,398 @@ definition {
+		}
+
+		task ("Then a permissions error is shown and no log is downloaded") {
+			AssertTextPresent(
+				locator1 = "//body",
+				value1 = "You don't have authorization to view this page.");
 
-		// TODO LRQA-75856
+			DownloadTempFile();
 
+			DMDocument.assertNoFileNameFromTempFolder(fileName = "${logName}");
+		}
 	}
 
 	@description = "A non-administrative user should get a permissions denied error when trying to access the UI to get company logging information"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test GetCompanyLogsAsGuest {
+		task ("Given a new user") {
+			JSONUser.addUser(
+				userEmailAddress = "userea@liferay.com",
+				userFirstName = "userfn",
+				userLastName = "userln",
+				userScreenName = "usersn");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "userea@liferay.com");
+		}
+
+		task ("When the user tries accessing the company logging UI to download a specific company log") {
+			PortalInstances.openVirtualInstancesAdmin();
+
+			var companyId = TestUtils.getElementTextContent(element = "//tr[contains(.,'liferay.com')]/td[1]//a");
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "o/company-log");
+
+			var logName = TestUtils.getElementTextContent(element = "//h1[text()='liferay.com']//following::a[1]");
+
+			var logSize = StringUtil.extractLast("${logName}", "log");
+
+			var logName = StringUtil.replace("${logName}", "${logSize}", "");
+
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfn userln");
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "o/company-log/${companyId}/${logName}?download=true");
+		}
+
+		task ("Then a permissions error is shown and no log is downloaded") {
+			AssertTextPresent(
+				locator1 = "//body",
+				value1 = "You don't have authorization to view this page.");
+
+			AssertTextPresent(
+				locator1 = "//body",
+				value1 = "HTTP ERROR 403");
 
-		// TODO LRQA-75857
+			DownloadTempFile();
 
+			DMDocument.assertNoFileNameFromTempFolder(fileName = "${logName}");
+		}
 	}
 
 	@description = "If there are no logs available for a company, there should be some kind of message to notify the user when they try to check the available log names"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test GetEmptyCompanyLogs {
+		property test.name.skip.portal.instance = "CompanyLogging#GetEmptyCompanyLogs";
+
+		task ("Given all company logs are deleted") {
+			var logsDir = PropsUtil.get("liferay.home.dir.name");
 
-		// TODO LRQA-75858
+			var logsDir = "${logsDir}/logs/companies";
+
+			FileUtil.delete("${logsDir}");
+		}
 
+		task ("When the user navigates to the company logging UI access point") {
+			Navigator.openWithAppendToBaseURL(urlAppend = "o/company-log");
+		}
+
+		task ("Then a message is shown that indicates no logs are available") {
+			AssertElementPresent(
```

Comment:
Since we will likely never re-use this path again (This path just needs to get the message content from the UI), you can hard code the locator into here. Creating a path that will not be re-used is like creating a function that will not be re-used